### PR TITLE
RnD update

### PR DESCRIPTION
**Pull request by Core0verload**
- DA menu shows current research levels. Small but really useful thing.
- You can sell research data disks and maxed reliablity design disks for supply points in cargo.
- Syndicate intel sell cost in cargo buffed to 250. Same cost as 50 plasma stack.
- You can print cyberimplants in updated mech fab.
- Added low-tech cyber implant - welding shield. It's cheap, it protects you from welders and flashes, but you need surgery to install it. Mostly pointless until #11058 would allow you to install more than one implant in eyes.
- Implanter and empty implant case added to protolathe.
- Added two syndie implants to protolathe. Freedom is Illegal 5, Adrenal is Illegal 6.

Tech cost is calculated as sum of all the tech levels cost except level 1. Tech level: 5 \* level \* multiplier, multiplier is 4 for illegal, 3 for plasma, 2 for bluespace, 1 for everything else. Materials 7 tech disk would be 5*(2+3+4+5+6+7) = 135 supply points.

### CODE REVIEW

**Comment by Jalleo** [comment]
actually that is probably the most useful implant I have heard of

**Comment by GunHog** [comment]
> DA menu shows current research levels. Small but really useful thing.

This alone makes this PR merge-worthy!
> You can sell research data disks and maxed reliablity design disks for supply points in cargo.

Fantastic! This can facilitate actual trade between the two departments rather than it being mostly one way! (Sell techs for conveyors and combat shotties)

>Syndicate intel sell cost in cargo buffed to 250. Same cost as 50 plasma stack.

I do not know what this is.
>You can print cyberimplants in updated mech fab.

No strong opinion. Good update, though.
> Added low-tech cyber implant - welding shield. It's cheap, it protects you from welders and flashes, but you need surgery to install it. Mostly pointless until #11058 would allow you to install more than one implant in eyes.

Overshadowed by welding gas mask, but I can see some people getting it for the sake of wanting to be Adam Jensen.

>Implanter and empty implant case added to protolathe.

I assume the implanter is needed when printing the Syndie stuff, but what is the point of an empty implant case?

>Added two syndie implants to protolathe. Freedom is Illegal 5, Adrenal is Illegal 6.

Absolutely delicious! :+1:

**Comment by Core0verload** [comment]
@Jalleo, it isn't gated behind syndie or alien research levels, so it's most useful eye implant until you get your biotech 7. It's EMP-proof too - engineers wouldn't be blinded by singulo.

@GunHog, empty implant case is useful in surgery (since #10957) when you want to extract an implant without destroying it.

**Comment by Jalleo** [comment]
@GunHog the files that traitors can spawn with those are the ones which can be sent to centcom for lots of points. Its a reason for security to pass em to cargo

**Comment by Miauw62** [comment]
dont do anything with reliability, the system is weird and confusing and shitty and should really be removed. getting max reliability is really easy. selling designs is cool, though. now we just need more stuff for cargo to buy :V

**Comment by Core0verload** [comment]
Reliability is bad, but when default reliability for all designs would be >= 100, designs selling would just harmlessly stop working. So my pull is reliability-remove friendly.

**Comment by GunHog** [comment]
Cargo needs a better interface and system. It simply is too clunky and takes too long to get things - hence why all the cargo players are lazy and other departments only go to them if they must (Security and Science)

**Comment by Aloraydrel** [comment]
This PR makes me happy

**Comment by xxalpha** [comment]
Don't put cyber implants in robotics fabricator.

Welding shield eyes makes no sense, but whatever.

File: code/modules/research/designs/medical_designs.dm [hunk 1]
```diff
@@ -111,15 +111,27 @@
...
 	name = "Medical HUD implant"
 	desc = "These cybernetic eyes will display a medical HUD over everything you see. Wiggle eyes to control."
 	id = "ci-medhud"
 	req_tech = list("materials" = 6, "programming" = 4, "biotech" = 4)
 	build_type = PROTOLATHE | MECHFAB
+	construction_time = 50
 	materials = list(MAT_METAL = 200, MAT_GLASS = 200, MAT_SILVER = 500, MAT_GOLD = 500)
 	build_path = /obj/item/organ/internal/cyberimp/eyes/hud/medical
-	category = list("Medical Designs")
+	category = list("Misc", "Medical Designs")
```
Comment (xxalpha) [inline code/modules/research/designs/medical_designs.dm]: Don't put them in two menus, that will be confusing?

**Comment by Core0verload** [comment]
@xxalpha, why? Most of cyberimplants and robotic are installed in robotics anyway. And I didn't removed implants from protolathe.

And if you view the code, implants being in mechfab was totally intended. They had MECHFAB buildflag, but they had no mechfab category.

**Comment by xxalpha** [comment]
Oh I see.

Carry on.

**Comment by Core0verload** [comment]
@xxalpha, in #11058 there are eyes HUD implants, eyes vision implants and eyes shield implants. You can install one HUD implant, one vision implant and one shield implant.

**Comment by xxalpha** [comment]
Still more power with no added drawbacks. The welding shield doesn't even have an emp_act().

**Comment by Core0verload** [comment]
That's because I wanted to make that shield useful for engineers. Engineering means singulo, singulo means EMP.

**Comment by Miauw62** [comment]
eh, add a minor emp_act().
singulo doesnt emp all the time and gives weak emp so a 2 second blind wouldnt be bad, really, and a 5 second blind on heavy emp would be ok.

**Comment by xxalpha** [comment]
Well cybernetic welding shielding makes no sense either, yet they exist.

And eye implant code isn't made for cumulative drawbacks, so you shouldn't be able to implant more than once.

**Comment by Core0verload** [comment]
Why drawbacks for everything? Finding a surgeon to install that implant is not so easy.

**Comment by xxalpha** [comment]
Because they don't occupy inventory space, are concealed and give you super powers.

**Comment by Core0verload** [comment]
Being protected from shitty welding eye ache is superpower?

**Comment by Miauw62** [comment]
its pretty powerful yeah because the only other places you can get those are hardsuits or limit your view.

**Comment by Core0verload** [comment]
And why welding shields must always limit your view? It was a shitty change by Erro afaik, welding mask was just making everything around you really dark.

**Comment by Jalleo** [comment]
I myself am a avid engineering player but I have to say this without any downsides will become a seriously big buff to security/gangs/revs/whatever. If you want to remove the other downsides make it vulnerable to the flash bangs and flashes so people can only use it for welding and nothing else.

**Comment by Lumipharon** [comment]
Most of this is ok, but the syndie implants? yeah please no.
Implants stack now - I can be traitor sci with infinite adrenal (which heals on top of the stun break, keep in mind), which is pretty damn close to god mode.

**Comment by Core0verload** [comment]
Seriously big buff? WHAT!?

You cannot just inject youself with implanter and be flashproof. You need someone who will surgically insert that thing into your eyes. And if flash immunity is really such a big buff, why do we keep goddamn STUN IMMUNITY IMPLANT in our nerfed-to-the-bottom RnD? And stunrevolvers, advanced eguns, some other things ranging from "mildly OP" to "OP OP OP OP NERF IT NOW", including goddamn COMBAT MECHS you don't even need sec access to ride.

@Lumipharon, don't worry. Adrenal injects you with omnizine for healing. 3-4 injections in a row and you are overdosed and dying. Plus it stacks only up to 2x uses from initial. Same with uncuff implant, but without overdose. And you need some traitor stuff to get your Illegal 5.

**Comment by Jalleo** [comment]
It is because it is cheap not because of it needing surgery you could technically do three five surgeries at the same time if you have the bed sheets and stuff ready for you.
It only costs like two metal and glass sheets for that amount if even that.

**Comment by Core0verload** [comment]
Then what material reqs it must have to be balanced? 

I just cannot balance things in SS13 because every time I try I realise there was no such thing as "balance" in the first place, and infinite nerfs, buffs and other beakers splashing didn't made anything better.

**Comment by Jalleo** [comment]
the best way is either to make it weak to flash stuff or do the visor style thing that the goggles and helmet and that have and reduce the mobs vision when its in use.
I do admit there is no perfect balance we don't try for that we just try to make it kind of balanced.

**Comment by Core0verload** [comment]
Making it weak to flashes means adding some snowflake code to welders. Adding vision impairment means touching that awful HUD code.

I think we can just add it as is. If it would be overused, report it to me and I will nerf it to require high tech, diamonds or such.

**Comment by PKPenguin321** [comment]
i actually think welding eye implant is a neat idea and i dont see it messing with balance that much except for a few edge cases in rev (but if the revs figure out they can't convert some scientist they'll just kill them anyways so it doesnt matter)

you guys are panicking a bit too much over practically nothing

**Comment by whodaloo** [comment]
can you make it so we can build welding goggles

**Comment by Lumipharon** [comment]
you can making a welding gas mask already with like, almost no research. If someone has the time to research, make and implant a welding mask into their eyeballs, good for them. 

Literally the only advantage over a welding gas mask is like, fuck I unno, someone stripping you naked then trying to flash you? A rev trying to flash you but suddenly flash protection?

Core, even if you can't spam it 20 times in 20 seconds, you still have a huge advantage over everyone else - 3 adrenal implants is like, 9 stun breaks + huge heals, and you can just carry a box of extra implanters around.

**Comment by TheVekter** [comment]
All hail based C0re

**Comment by Iamgoofball** [comment]
I highly suggest axing reliability completely. It's literally rng "sorry print a new one

**Comment by Iamgoofball** [comment]
also stop caring about balance

care about what is fun and what is not fun

**Comment by lordpidey** [comment]
My understanding of reliability is that there was no RNG component.

Your destructive analyzer simply has a minimum required reliability.  Below that, and the item doesn't give research.  (Not sure if it still gives small amounts of minerals)

**Comment by Kelenius** [comment]
Yea, I'm with goof here, reliability is bugged, unfinished, and unfun.

**Comment by Lumipharon** [comment]
Whelp, time for infinite unstunnable,uncuffable traitor sci.

**Comment by PKPenguin321** [comment]
i'll bet you 20 dollars that will not happen

**Comment by Lumipharon** [comment]
It takes precisely 1tc to unlock those implants. Please explain why any traitor would NOT stock up on adrenal and freedom implants given the chance?

**Comment by PKPenguin321** [comment]
because our traitors are shit

**Pull request merged**